### ISSUE TITLE
Wrong result running example textclassification

### ISSUE THREAD

**Issue by roywei** [NVIDIA/DIGITS#906]
Hi, I was following this example here: [text classification](https://github.com/NVIDIA/DIGITS/tree/master/examples/text-classification)

I was using a AWS g2.8xlarge machine, and followed the same steps using same data set(DBPedia) and same model [here](https://github.com/NVIDIA/DIGITS/blob/master/examples/text-classification/text-classification-model.lua), however I get the following result:
![image](https://cloud.githubusercontent.com/assets/8022184/16858035/db755d38-49d8-11e6-9148-60239b39a9c1.png)

I have tried to run longer time (till epch 10), still the same result.
May I know which of my steps might be wrong?

Thanks a lot.

**Comment by gheinrich** [comment]
Hello, did you follow the instructions for [model creation](https://github.com/NVIDIA/DIGITS/tree/v4.0.0-rc.3/examples/text-classification#model-creation) carefully (e.g. mean subtraction set to none, etc.)? The model generally starts learning something useful after epoch 0.5 (at which point you should see the accuracy above 20% and going up steadily).

**Comment by roywei** [comment]
Thank you so much for your quick reply!
Yes I did (also set 15 epoch, 0.25 validation, exponential decay learning rate), following is my setting, and you can also take a look [here](http://ec2-54-183-220-6.us-west-1.compute.amazonaws.com/)
![image](https://cloud.githubusercontent.com/assets/8022184/16878659/46f9713c-4a62-11e6-9289-0911b1689326.png)
![image](https://cloud.githubusercontent.com/assets/8022184/16878682/533f4764-4a62-11e6-9c7b-81e1b5fc4312.png)
![image](https://cloud.githubusercontent.com/assets/8022184/16878699/636bad3a-4a62-11e6-96e4-61c0c5ee22d8.png)

**Comment by gheinrich** [comment]
Oh I think you're using an old version of DIGITS which has a bug in the exponential decay learning rate policy so the learning rate becomes ridiculously small very quickly. If you can upgrade to a newer version that would be nice, otherwise try using the default learning rate policy.

**Comment by roywei** [comment]
Thank you! It's indeed the exponential rate problem.
Another quick question, when building custom model from torch, it says multi-gpu is not supported, but I found [getting started torch page](https://github.com/NVIDIA/DIGITS/blob/master/docs/GettingStartedTorch.md#multi-gpu-training) it's supported.
![image](https://cloud.githubusercontent.com/assets/8022184/16884518/bae00cb6-4a7e-11e6-96e4-42995e05e474.png)

I installed digits from pre-built packages for ubuntu, shall I install from source to use the latest version? Thanks

**Comment by lukeyeager** [comment]
The latest packages are for 3.0 ([tagged Jan 21](https://github.com/NVIDIA/DIGITS/tags)). We haven't released packages for 4.0 yet.

So yes, you'll need to install from source if you want a newer version.

**Comment by roywei** [comment]
Thank you so much for your help!

**Comment by bnascimento** [comment]
OK, I ended up with the same problem wIth Digits 3.0. Thing is no matter what you put on for learning rate policy, you always end up with the same convergence to 2.5 loss. 
Anyone else with the same problem?

**Comment by roywei** [comment]
build Digits from source instead of ubuntu package helps, this example is running fine with various learning rate policy, and with multi GPU support.
